galleon-pack/src/main/resources/layers/standalone/observability/layer-spec.xml

```diff
@@ -8,5 +8,4 @@
     <feature-group name="health"/>
     <feature-group name="metrics"/>
     <feature spec="subsystem.microprofile-config-smallrye"/>
-    <feature spec="subsystem.microprofile-opentracing-smallrye"/>
```

Comment:
@jfdenise Instead of complete separation, I'd prefer having observability having an optional dependency on open-tracing. So OT can be easily excluded.

Observability is a fairly broadly used concept and has been out there for a while now, so I don't think we should redefine it.

Reply:
@bstansberry , I agree. Will make changes to the PR.